### ISSUE TITLE
Failed prop type: Invalid prop `accessibilityRole` of value `tab` supplied to `TouchableOpacity`

### ISSUE THREAD

**Issue by venkatarajeshjakka** [satya164/react-native-tab-view#1000]
getting following **warning** 

I am using "react-native-tab-view": "^2.14.0",

Warning: Failed prop type: Invalid prop `accessibilityRole` of value `tab` supplied to `TouchableOpacity`, expected one of ["none","button","link","search","image","keyboardkey","text","adjustable","imagebutton","header","summary"].
- node_modules/expo/build/environment/muteWarnings.fx.js:27:24 in error
- ... 25 more stack frames from framework internals

**Comment by geiz** [comment]
If you are using Expo, try upgrading to the latest version of Expo (37) and then go through the installation process for react-native-tab-view again.

If you are not using Expo, try [https://github.com/facebook/react-native/issues/27894#issuecomment-586592279](https://github.com/facebook/react-native/issues/27894#issuecomment-586592279)

**Comment by hmisonne** [comment]
I had the same error message and I was able to fix it by using createBottomTabNavigator instead of createMaterialTopTabNavigator.

**Comment by Haimma** [comment]
> createMaterialTopTabNavigator

where? I didnt find it in the package

**Comment by satya164** [comment]
Closing since there's no error with latest version of the library and react-native
